### PR TITLE
FAQ for review

### PR DESCRIPTION
**Pull request by jschlyter**
Frequently asked questions written by @gabywh with edits from @jschlyter.

### CODE REVIEW

**Review by gabywh — APPROVED** [review]

File: FAQ.md [hunk 1]
```diff
@@ -1,3 +1,69 @@
...
+Please note that if the value is available, then it is required to be set, as per  [DICOM Type 2](http://dicom.nema.org/medical/dicom/current/output/html/part05.html#sect_7.4.3):
+
+> [... defines] Type 2 Data Elements that shall be included and are mandatory Data Elements. However, it is permissible
+> that if a Value for a Type 2 element is unknown it can be encoded with zero Value Length and no Value.
+> **If the Value is known the Value Field shall contain that value [...].
+> These Data Elements shall be included in the Data Set and their absence is a protocol violation.**
+
+## <a id="uvci_format"></a>Annex 2 specifies three different formats for certificate id (UVCI). Which one do I use?
+
+For _generating_ the certificate id: the choice of which option to choose from Annex 2 in the  [eHealthNetwork Vaccination Interoperability Guidelines](https://ec.europa.eu/health/sites/health/files/ehealth/docs/vaccination-proof_interoperability-guidelines_en.pdf) is delegated to the Member State.
```
Comment (martin-lindstrom) [inline FAQ.md:51]: No examples of UVCI:s?

Did I miss the discussion about which characters that should be included for Luhn mod N?

File: FAQ.md [hunk 2]
```diff
@@ -1,3 +1,69 @@
...
+
+The DGC Schema supports conformity to the proposed [EU legislation](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:52021PC0130) is intended to be a vehicle for serialization and de-serialization according to several differing Member State requirements.  For example, one Member State wishes to generate one QR code per [certificate type](#cert_types) and another may wish to generate a QR code combining two or three types of certificate. Both of these scenarios are supported by the DGC Schema. The business rules for that Member State, however, determine precisely which data the DGC schema will be populated with.
+
+## Why are the certificate type entries specified as an array?
+
+The DGC schema presented here allows for multiple of all entries (in essence: array of <type>). This is because, the DGC schema (1) is designed in such a way that if regulations require (or allow) more than one entry per type,  then there is no fundamental schema structure change, and (2) that the current regulation-conform implementation of only one entry can just be considered as an array of one element.
```
Comment (martin-lindstrom) [inline FAQ.md:65]: Add a sentence: "If more than one entry is included the list should be sorted with the most recent entry first". Or something like this? There are discussions where some state that only one entry should be included and other want to include several. In the cases several entries are included we will increase the chances of good interoperability if the most important entry (the most recent) is placed first in the list. Some verifiers maybe only reads the first entry.

**Review by martin-lindstrom — APPROVED** [review]
Approved, but check my comments/suggestions.

File: FAQ.md (hunk 1 above)
Comment (gabywh) [reply FAQ.md:51]: No, not missed. I'm waiting / blocked by this review so I can add the Luhn-Mod-N text + example (@jschlyter really wanted the text in the wiki for some reason so now I can't edit the wiki until this reviewed, approved and merged)

**Review by kruzikh — APPROVED** [review]

File: FAQ.md (hunk 2 above)
Comment (gabywh) [reply FAQ.md:65]: > "If more than one entry is included the list should be sorted with the most recent entry first"

Or last entry is most recent, also do-able.
Nothing in regulations or guidelines. I guess it is easy enough to sort on date if needed - and which date to sort on? Valid from? Valid until? Most recently entered? I guess us ordering internally is an option but has some (if limited) usefulness. Is this a blocking issue or can we take it up post v1.0.0 ?

**Comment by jschlyter** [comment]
merged to wiki

File: FAQ.md (hunk 2 above)
Comment (martin-lindstrom) [reply FAQ.md:65]: Let's leave it out for now ...

**Pull request closed without merging**